test/indexer/test_anvil.py

```diff
@@ -56,7 +56,7 @@ class AnvilIndexerTestCase(AnvilCannedBundleTestCase, IndexerTestCase):
     def bundle_fqid(cls,
                     *,
                     uuid,
-                    version,
+                    version='',
```

Comment:
Is this ever called with a version other than `''`? If not, I would just drop the parameter.

Reply:
This method is inherited from `IndexerTestCase`. The `version` parameter is not used for AnVIL tests, but it is used in other classes that inherit from `IndexerTestCase`. I thought it would be a problem to remove the parameter in this subclass but keep it in the superclass.

Reply:
Makes sense.